pkg/bosh/manifest/dns.go

```diff
@@ -21,19 +22,34 @@ import (
 	"code.cloudfoundry.org/quarks-utils/pkg/names"
 )
 
-//DomainNameService abstraction
+var (
+	boshDNSDockerImage = ""
+	clusterDomain      = ""
```

Comment:
Using a global is kind of icky, but :man_shrugging:

Reply:
I felt the same but didn't come with a better solution at the time. I just went with the current setup.